fake-koji/src/main/java/org/fakekoji/core/AccessibleSettings.java

```diff
@@ -128,6 +141,7 @@ public AccessibleSettings(
         this.projectMapping = new ProjectMapping(this);
         this.reportParams = reportParams;
         this.reportChartParams = reportChartParams;
+        JenkinsCliWrapper.setMainSettings(this);
```

Comment:
This is weird, but there should be only one settings in whole app. The AccessibleSettings should have had been singleton since origin